Mathlib/SetTheory/Cardinal/Arithmetic.lean

```diff
@@ -415,6 +415,15 @@ protected theorem ciSup_mul_ciSup (g : ι' → Cardinal.{v}) :
     (⨆ i, f i) * (⨆ j, g j) = ⨆ (i) (j), f i * g j := by
   simp_rw [Cardinal.ciSup_mul f, Cardinal.mul_ciSup g]
 
+theorem sum_eq_iSup_lift {f : ι → Cardinal.{max u v}} (hι : ℵ₀ ≤ #ι)
```

Comment:
Based on the surrounding lemmas, I think this should use `ciSup` in the name?

Reply:
As I understand it (or rather, as @YaelDillies told me), the convention is to use `iSup` rather than `ciSup` for concrete types.

Reply:
Or is the rule about where a `BddAbove` assumption is needed?

Reply:
The rule as I have understood it is that the `c` is used as disambiguation, and dropped when there is no ambiguity. Here there can't be an honest to god iSup, so ciSup means iSup

Reply:
I tend to agree it's a pretty weird rule